### ISSUE TITLE
Documentation error in the SequencesInDepth tutorial

### ISSUE THREAD

**Issue by marehr** [seqan/seqan#1185]
There is a typo in the code snippet of the documentation in
http://seqan.readthedocs.org/en/master/Tutorial/SequencesInDepth.html#overflow-strategies

The doc says:
```c++
String<Dna> dnaSeq;
// Sets the capacity of dnaSeq to 5.
resize(dnaSeq, 4, Exact());
// Only "TATA" is assigned to dnaSeq, since dnaSeq is limited to 4.
assign(str, "TATAGGGG", Limit());
std::cout << dnaSeq << std::endl;
// Use the default expansion strategy.
append(dnaSeq, "GCGCGC");
std::cout << dnaSeq << std::endl;
```

The following line uses an undefined variable. I assume, that you actual meant `dnaSeq` and not `str`.
```c++
assign(str, "TATAGGGG", Limit());
```

**Comment by esiragusa** [comment]
Will be fixed by #1173 #1174.

**Comment by esiragusa** [comment]
Fixed in e31f1f2 (#1197)
